Orange/data/domain.py

```diff
@@ -63,6 +75,19 @@ def __init__(self, source, destination):
             source.index(var) if var in source
             else var.compute_value for var in destination.metas]
 
+        def is_sparse(feats):
+            """ For a matrix to be stored in sparse, more than 2/3 of columns
+            should be marked as sparse and there should be no string columns
+            since Scipy's sparse matrices don't support dtype=object."""
```

Comment:
At least the closing triple quotes of a multi-line docstring should be in their own line (PEP257).
(for opening quotes try to mimic other docstrings nearby)

Reply:
Changed to match the style as in nerby docstrings.